React-Todo-list-main/src/Components/TodoWrapper.js

```diff
@@ -46,12 +56,13 @@ export const TodoWrapper = () => {
 
   /**
    * Setting the experiment and feature value
-   * 
+   *
    */
   useEffect(() => {
-    console.log(`Experiment Config: ${JSON.stringify(config)}`);
-    setSortingOrder(config.value.sort_order);
+    console.log(`Experiment Config: ${JSON.stringify(experimentConfig)}`);
+    setSortingOrder(experimentConfig.value.sort_order);
     setFeatureValue(value);
+    setDynamicValue(dynamicConfig.value);
```

Comment:
Dont have a state for the sort order and the dynamic config value and use a useEffect - just call `useConfig` and use that value directly